src/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.1.2"
+__version__ = "0.1.3+build.1"
```

Comment:
Why did you need to bump the build here as well?  Did you build a 0.1.3 AMI that had an error?

Reply:
I didn't- I thought that the first build should be `0.1.3+build.1` and when everything is working, it would end up at `0.1.3`, but based on your question, I'm thinking that's not correct.

Reply:
I do `0.1.3-rc.1` and `0.1.3-rc.2`, for example, for the staging AMI builds and then finalize that to `0.1.3` when everything is working and it's time to build the production AMI.

The build numbers are for when you don't change the code but rebuild the AMI, for example when freshening the AMIs for the monthly redeploy.

Reply:
Got it- I will try to remember to follow that same scheme in the future.

Reply:
> Got it- I will try to remember to follow that same scheme in the future.

I'll be watching. :eyes:

Reply:
Proof: https://github.com/cisagov/pca-gophish-composition-packer/pull/33/commits/0196468386b8679034579782b5c1f9d5380d95c8

Reply:
[Behold!](https://github.com/cisagov/pca-gophish-composition-packer/releases/tag/v0.1.3%2Brc.1)